riocli/v2client/client.py

```diff
@@ -339,7 +339,7 @@ def list_static_routes(
         while True:
             params.update({
                 "continue": offset,
-                "limit": 50,
+                "limit": 500,
```

Comment:
suggestion: Consider removing the limit altogether. That way, the server will inject whatever is the configured limit. We won't have to update it again we can just change server parameters and it will just work.